mgr/manager.go

```diff
@@ -23,28 +24,39 @@ const DefaultInterval = time.Hour
 // Manager. This defaults to 72 hours.
 const DefaultBefore = time.Hour * 72
 
+// MgrSpecOptions is a compatibility shim for mapping old manager configurables into
+// the common names used by SpecOptions.
+type MgrSpecOptions struct {
+	cert.ParsableSpecOptions
+
+	// OldServiceManagerField is the old yaml configurable name used for this.
+	OldServiceManagerField string `yaml:"service_manager"`
+
+	// OldRemoteName is the old yaml configurable name used for this.
+	OldRemoteField string `yaml:"default_remote"`
+}
+
+// FinalizeSpecOptionParsing should be invoked to transfer fields from old location names
+// to new location names for compatibility.
+func (m *MgrSpecOptions) FinalizeSpecOptionParsing() {
+	if m.OldServiceManagerField != "" {
```

Comment:
It might be helpful to warn about usage of the deprecated configuration options. Another thing to consider is what should be done if both the old and new keys are specified. Currently, it looks like the old key would take precedence if it exists. But, this is probably an edge case and won't actually happen (and that behavior sounds fine), so feel free to disregard.

Reply:
I'll update the PR with that shortly.